File: examples/stock/orderBook/go/orderBookHistory.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/stock"
)

// Get the historic order book and each trade by timestamp
func OrderBookHistory() {
	stockService := stock.NewStockService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := stockService.OrderBook(&stock.OrderBookRequest{
		Date:  "2020-10-01",
		End:   "2020-10-01T11:00:00Z",
		Limit: 3,
		Start: "2020-10-01T10:00:00Z",
		Stock: "AAPL",
	})
	fmt.Println(rsp, err)
}

File: examples/js/stock/orderBook/orderBookHistory.js
const { StockService } = require("m3o/stock");

// Get the historic order book and each trade by timestamp
async function orderBookHistory() {
  let stockService = new StockService(process.env.MICRO_API_TOKEN);
  let rsp = await stockService.orderBook({
    date: "2020-10-01",
    end: "2020-10-01T11:00:00Z",
    limit: 3,
    start: "2020-10-01T10:00:00Z",
    stock: "AAPL",
  });
  console.log(rsp);
}

orderBookHistory();
